### PR TITLE
Agdex

### PR DESCRIPTION
**Pull request by liuxieric123**
Agdex' TVL needs to include another small part of the assets locked in the user's account, so this PR was made

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/agdex exports TVL: 
        
 
 
    aptos                     4.52 k
    
    total                    4.52 k

File: projects/helper/http.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ async function post(endpoint, body, options) {
     return data
   } catch (e) {
     sdk.log(e.message)
+    console.log(e)
```
Comment (g1nt0ki) [inline projects/helper/http.js:47]: please remove this line

**Comment by llamatester** [comment]
The adapter at projects/agdex exports TVL: 
        
 
 
    aptos                     4.46 k
    
    total                    4.46 k

File: projects/helper/http.js (hunk 1 above)
Comment (ryanlin-tech) [reply projects/helper/http.js]: done

**Pull request merged**